hw02-gradle/src/main/java/ru/inno/java/DIYArrayList.java

```diff
@@ -0,0 +1,261 @@
+package ru.inno.java;
+
+import java.util.*;
+
+public class DIYArrayList<E> implements List<E> {
+
+    private int INITIAL_CAPACITY = 16;
```

Comment:
Если это константа, то `final static`

```suggestion
    private final static int INITIAL_CAPACITY = 16;
```